src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTreeUI.java

```diff
@@ -3280,6 +3280,7 @@ public Rectangle getNodeDimensions(Object value, int row,
                      expanded, treeModel.isLeaf(value), row,
                      false);
                 if(tree != null) {
+                    rendererPane.removeAll();
                     // Only ever removed when UI changes, this is OK!
```

Comment:
The comment no longer applies—you remove all the components from `rendererPane` on the line above.

Reply:
This is the only important thing that prevents me from approving this PR. The comment is misleading now.

Reply:
OK, modified